python/open3d/__init__.py

```diff
@@ -34,57 +34,86 @@
 # https://github.com/dmlc/xgboost/issues/1715
 import os
 import sys
+
 os.environ['KMP_DUPLICATE_LIB_OK'] = 'True'
 from ctypes import CDLL as _CDLL
 from ctypes.util import find_library as _find_library
 from pathlib import Path as _Path
-import warnings
+import logging
 
 from open3d._build_config import _build_config
+
+
+def _load_cdll(path):
+    """
+    Wrapper around ctypes.CDLL to take care of Windows compatibility.
+    """
+    path = _Path(path)
+    if not path.is_file():
+        raise FileNotFoundError(f"Shared library file not found: {path}.")
+
+    if sys.platform == 'win32' and sys.version_info >= (3, 8):
```

Comment:
From the [linked Python issue](https://bugs.python.org/issue42114), looks like this is fixed in 3.9, 3.10 and onwards. Should we only check for 3.8 here?